### PR TITLE
Improve tests

### PR DESCRIPTION
**Pull request by Demetriex**
Organize test so its easier to add tests in the future

### CODE REVIEW

**Comment by Demetriex** [comment]
https://github.com/greyli/bootstrap-flask/blob/21853a9b38a68ad0b9bb402365617d3402fecb47/flask_bootstrap/__init__.py#L98-L99

 We should also add test when `with_jquery` and `with_popper` args are false

**Comment by greyli** [comment]
Thank you very much! I think move the tests inside the `macros` folder to the root of the `tests` folder would be better.

File: tests/test_themes.py [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+    'solar',
+    'spacelab',
+    'superhero',
+    'united',
+    'yeti'
+]
+
+
+class TestThemes:
+    @pytest.mark.parametrize("theme", themes)
```
Comment (greyli) [inline tests/test_themes.py:32]: In this line and related lines, the single quotation marks should be used in order to consistent with others.

Comment (Demetriex) [reply tests/test_themes.py]: Thanks for pointing that out, i have updated it on the new commit

**Comment by greyli** [comment]
Merged, thanks!

**Pull request merged**